src/WebService/Client.php

```diff
@@ -163,7 +172,10 @@ public function post(string $service, string $path, array $input): ?array
 
     public function get(string $service, string $path): ?array
     {
-        $request = $this->createRequest($path);
+        $request = $this->createRequest(
+            $path,
+            ['Content-Type: application/json']
```

Comment:
I don't think we want to set the `Content-Type` here. GET requests generally do not have a body.